package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "retack-sdk-app-observer",
-  "version": "1.0.4",
+  "version": "1.0.3",
```

Comment:
the sdk version should increase

Reply:
The published version of the SDK is 1.0.2, but in a previous commit, it was mistakenly changed to 1.0.4. I have refactored and corrected it to 1.0.3.